{
  "schema_version": "1.4.0",
  "id": "GHSA-hpqq-hrxq-vc4x",
  "modified": "2025-06-26T18:31:24Z",
  "published": "2025-06-20T21:32:06Z",
  "aliases": [
    "CVE-2025-46158"
  ],
  "details": "An issue in redoxOS kernel before commit 5d41cd7c allows a local attacker to cause a denial of service via the `setitimer` syscall",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2025-46158"
    },
    {
      "type": "WEB",
      "url": "https://github.com/Marsman1996/pocs/tree/master/redox/CVE-2025-46158"
    },
    {
      "type": "WEB",
      "url": "https://gitlab.redox-os.org/redox-os/kernel/-/issues/164"
    },
    {
      "type": "WEB",
      "url": "https://gitlab.redox-os.org/redox-os/kernel/-/merge_requests/386"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-369"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2025-06-20T18:15:28Z"
  }
}